hbase-hadoop-compat/src/main/java/org/apache/hadoop/hbase/replication/regionserver/MetricsReplicationSourceSource.java

```diff
@@ -78,4 +77,6 @@
   void incrFailedRecoveryQueue();
   void setOldestWalAge(long age);
   long getOldestWalAge();
+  void setPeerZkConnectionFailures(boolean success);
```

Comment:
This is weird. It should be an increment function. See above incrFailedRecoveryQueue as example.